### PR TITLE
Fix roundtripping with language tags

### PR DESCRIPTION
**Pull request by vcharpenay**
Implemented solution: turn `dct:title` into `td:title` and `td:titleInLanguage` (same for `dct:description`).

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/wot-thing-description/pull/1308.html" title="Last updated on Dec 5, 2021, 7:35 PM UTC (c2d9f0c)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/wot-thing-description/1308/7343d5a...c2d9f0c.html" title="Last updated on Dec 5, 2021, 7:35 PM UTC (c2d9f0c)">Diff</a>

### CODE REVIEW

**Comment by vcharpenay** [comment]
@murloc6 could you review the changes?

File: ontology/td.ttl [hunk 1]
```diff
@@ -1,5 +1,6 @@
 @prefix : <https://www.w3.org/2019/wot/td#> .
 @prefix dc: <http://purl.org/dc/elements/1.1/> .
+@prefix dct: <http://purl.org/dc/terms/> .
```
Comment (ethieblin) [inline ontology/td.ttl:3]: dcterms: prefix was already defined L10

File: ontology/jsonschema.template.html [hunk 2]
```diff
@@ -142,8 +143,8 @@ <h3>Interpreting JSON Schema as JSON-LD 1.1</h3>
         "readOnly": { "@type": "xsd:boolean" },
         "format": { "@type": "xsd:string" },
         "required": { "@type": "xsd:string", "@container": "@set" },
-        "title": { "@type": "xsd:string" },
-        "description": { "@type": "xsd:string" }
+        "title": { "@id": "dtc:title", "@type": "xsd:string" },
```
Comment (ethieblin) [inline ontology/jsonschema.template.html:146]:
```suggestion
        "title": { "@id": "dct:title", "@type": "xsd:string" },
```

File: ontology/jsonschema.template.html [hunk 3]
```diff
@@ -142,8 +143,8 @@ <h3>Interpreting JSON Schema as JSON-LD 1.1</h3>
...
+        "description": { "@id": "dtc:description", "@type": "xsd:string" }
```
Comment (ethieblin) [inline ontology/jsonschema.template.html:147]:
```suggestion
        "description": { "@id": "dct:description", "@type": "xsd:string" }
```

**Comment by Murloc6** [comment]
Thank you for this contribution.
Except for the relation names, this PR is a good contribution, and it will definitively help for round tripping in our implementation.
Thank you :)

**Comment by vcharpenay** [comment]
thank you, @ethieblin, for the review. See last commit. The PR is ready to be merged, as far as I'm concerned.

**Review by ethieblin — APPROVED** [review]

**Review by sebastiankb — APPROVED** [review]

**Pull request merged**